scripts/wrapper/withdraw.js

```diff
@@ -207,7 +207,7 @@ module.exports = function (web3, artifacts) {
 
     log("Started building withdraw transaction.")
     const transactionPromise = buildWithdrawAndTransferFundsToMaster(argv.masterSafe, withdrawals)
-
+    console.log(transactionPromise)
```

Comment:
Was this meant to be left here? If, so I would prefer if you used the logger defined at the beginning of the function that only logs when `debug == true`